### ISSUE TITLE
[SRS] EOM methods type

### ISSUE THREAD

**Issue by gabrielasd** [gabrielasd/eomee#43]
![fix_section5 1](https://user-images.githubusercontent.com/26312264/101522645-84537f00-3955-11eb-803e-30df585a1b34.png)

**Comment by gabrielasd** [comment]
Hello Dr. @smiths 
When you mention defining a type, would it be something like a variable referring to a set of all EOM methods in the document? Like eomT = {IP_EOM, EA_EOM, ...}?

**Comment by smiths** [comment]
Yes, that is what I mean.  This would be implemented as an enum in Python.  In the math of the MIS you would define it as you have above.
